hwilib/descriptor.py

```diff
@@ -172,6 +204,17 @@ def __init__(
     def to_string_no_checksum(self) -> str:
         return "{}({},{})".format(self.name, self.thresh, ",".join([p.to_string() for p in self.pubkeys]))
 
+    def expand(self, pos: int) -> "ExpandedScripts":
+        m = (self.thresh + 0x50).to_bytes(1, "big") if self.thresh > 0 else b"\x00"
```

Comment:
might want to throw when self.thresh is larger than 16 since we're dealing with single pushes only for now

Reply:
Changed to handle thresholds larger than 16